proteus/SpatialTools.py

```diff
@@ -767,7 +767,7 @@ def __init__(self, domain, dim=(0., 0.), coords=(0., 0.), barycenter=None):
                         self.BC['y+'],
                         self.BC['x-']]
         # self.BC = BCContainer(self.BC_dict)
-        self.It = L**2+H**2/12
+        self.It = (L**2+H**2)/12
```

Comment:
Yes, it looks like the brackets were forgotten. It could affect results but I don't think this affected previous results in my case as I would set the mass moment of inertia directly.